_topic_map.yml

```diff
@@ -42,6 +42,44 @@ Distros: openshift-acs
 Topics:
 - Name: Service description
   File: rhacs-cloud-service-service-description
+- Name: Getting started with RHACS Cloud Service
+  File: getting-started-rhacs-cloud-ocp
+- Name: Setting up RHACS Cloud Service with Red Hat Cloud secured clusters
```

Comment:
```suggestion
- Name: Installing RHACS Cloud Service with Red Hat Cloud secured clusters
```

Reply:
I was asked to use "setting up" instead of "installing" because Cloud Service, itself, isn't something that customers install on anything. (They only set up the ACS instance and install secured cluster resources on clusters.)